### ISSUE TITLE
SWF websocket fallback script

### ISSUE THREAD

**Issue by Marak** [laktek/realie#2]
would it be possible to add a swf fallback if websockets arent available?

i believe there are libraries around for doing this.

**Comment by laktek** [comment]
Yeah there are. Node.js web-socket module is also compatible with swf fallbacks.

Will consider on implementation in the latter stage of the project.
